Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -2836,7 +2836,8 @@ void FrameLoader::checkLoadCompleteForThisFrame(LoadWillContinueInAnotherProcess
         if (RefPtr page = m_frame->page()) {
             if (isBackForwardLoadType(loadType())) {
                 // Reset the back forward list to the last committed history item at the top level.
-                item = page->mainFrame().history().currentItem();
+                if (auto* localMainFrame = dynamicDowncast<LocalFrame>(page->mainFrame()))
```

Comment:
RefPtr?

Reply:
smart pointer guidelines say we can use raw pointers when calling trivial and inlined functions